websites/M/Memrise/presence.ts

```diff
@@ -0,0 +1,254 @@
+const presence = new Presence({ clientId: "1095377958241304586" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/emLcjZT.png",
+	Searching = "https://i.imgur.com/OIgfjTG.png",
```

Comment:
Please use 512x512, I will update the docs page in a sec, I think the README in this repository already has that it must be exactly 512x512.

Reply:
Just like Darkvillager said, Thats the resourses standards height & width. Unless the default icons had beeen changed too.

Reply:
Oh alright, yea didn't know they weren't, I have updated them in https://github.com/PreMiD/Presences/commit/28cd6f6f8f5e7318cd109de4ecc5552c93da4683